src/bg_mpl_stylesheets/tests/test_colors.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from bg_mpl_stylesheets.colors import Colors
+
+
+@pytest.mark.parametrize(
+    "hex, expected_hex",
+    [
+        (Colors.bg_blue, "#0B3C5D"),
+        (Colors.bg_red, "#B82601"),
+        (Colors.bg_green, "#1C6B0A"),
+        (Colors.bg_light_blue, "#328CC1"),
+        (Colors.bg_light_grey, "#A8B6C1"),
+        (Colors.bg_yellow, "#D9B310"),
+        (Colors.bg_brown, "#6C5050"),
+        (Colors.bg_burgundy, "#76323F"),
+        (Colors.bg_olive_green, "#626E60"),
+        (Colors.bg_muted_olive, "#918770"),
+        (Colors.bg_beige, "#C09F80"),
+        (Colors.bg_grey, "#B0B0B0FF"),
+    ],
+)
+def test_color_values(hex, expected_hex):
+    # Test the hex values of the colors
+    assert hex == expected_hex
+
+
+@pytest.mark.parametrize(
```

Comment:
Here, `get_color_name(hex)` could instead return a string of `bg-color` instead of `bg_color` but the user probably would be storing the color name string and use that to get the hex code (maybe)

Keep all `lower_cased` would be a simple solution for all. @sbillinge Do you agree?